iOS Client/Actions/MBActions.swift

```diff
@@ -51,3 +51,9 @@ struct LoadedPodcasts: Action {
 struct SelectedPodcast: Action {
     var podcast: MBPodcast
 }
+
+struct ResumePodcast: Action {
+}
+
+struct PausePodcast: Action {
+}
```

Comment:
Just a thought, but I wonder if we could get away with a single action that just toggles it

Reply:
Yeah we might be able to get away with just that, good call